libs/next_gen_ui_agent/types.py

```diff
@@ -212,6 +212,30 @@ class Rendition(BaseModel):
     content: str
 
 
+class UIBlock(BaseModel):
+    """UI Block model with all details"""
+
+    id: str
+    rendering: Optional[Rendition] = None
+    "Rendering of UI block"
+
+
+class MCPGenerateUIInput(BaseModel):
```

Comment:
Why are these MCP specific types in the core? Or ff they are not MCP specific but useful for other AI bindings also, then rename them not to contain MCP in the name please.